mkdocs.yml

```diff
@@ -0,0 +1,9 @@
+site_name: osgeonepal
+site_url: https://hem2058.github.io/
```

Comment:
This one need to be changed to osgeonepal.github.io

Reply:
Okay dai, I will try to deploy on osgeonepal page also.